mm/2s2h/Enhancements/Audio/Sfx/LinksVoicePitchMultiplier.cpp

```diff
@@ -0,0 +1,37 @@
+#include <libultraship/bridge/consolevariablebridge.h>
+#include "2s2h/GameInteractor/GameInteractor.h"
+#include "2s2h/ShipInit.hpp"
+
+extern "C" {
+#include "variables.h"
+}
+
+// For Link's voice pitch SFX modifier
+static f32 freqMultiplier = 1;
+
+#define CVAR_NAME "gAudioEditor.LinkVoiceFreqMultiplier.Enable"
+#define CVAR CVarGetInteger(CVAR_NAME, 0)
+
+void RegisterLinksVoicePitchMultiplier() {
+    COND_VB_SHOULD(VB_LINK_VOICE_PITCH_MULTIPLIER, CVAR, {
+        *should = true;
+        Player* player = GET_PLAYER(gPlayState);
+        u16 sfxId = *va_arg(args, u16*);
+
+        if (sfxId >= NA_SE_VO_LI_SWORD_N && sfxId <= NA_SE_VO_DEMO_394 || sfxId == NA_SE_PL_TRANSFORM_VOICE) {
+
```

Comment:
At this point in your code within the conditional, we know we're processing a Link voice sfx, so we know not to do the vanilla behavior, thus we can set this here.
```suggestion
            *should = false;
```